code/game/objects/structures/mirror.dm

```diff
@@ -124,7 +124,7 @@
 		if("Body")
 			if(organ_warn)
 				to_chat(user, "<span class='boldwarning'>Using the mirror will destroy any non biochip implants in you!</span>")
-			var/list/race_list = list("Human", "Tajaran", "Skrell", "Unathi", "Diona", "Vulpkanin", "Nian", "Slime People", "Grey", "Drask", "Kidan")
+			var/list/race_list = list("Human", "Tajaran", "Skrell", "Unathi", "Diona", "Vulpkanin", "Nian", "Grey", "Drask")
```

Comment:
will this prevent wizards being slimes and kidan?

Reply:
Probably looking over it again, if so, this also means the karma pr blocked out vox and plasmamen from wizards.

Reply:
fixed